mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -357,40 +361,58 @@ void updateAnnotationsShouldThrowExceptionIfRequestCreateNewButMailboxIsOverLimi
 
     @Nested
     class EventTests {
-        @Test
-        void deleteMailboxShouldFireMailboxDeletionEvent() throws Exception {
-            assumeTrue(mailboxManager.hasCapability(MailboxCapabilities.Quota));
+        private final QuotaRoot quotaRoot = QuotaRoot.quotaRoot("#private&USER_1", Optional.empty());
+        private EventCollector listener;
+        private MailboxPath inbox;
+
+        @BeforeEach
+        void setUp() throws Exception {
             session = mailboxManager.createSystemSession(USER_1);
+            inbox = MailboxPath.inbox(session);
 
-            EventCollector listener = new EventCollector();
+            listener = new EventCollector();
             mailboxManager.addGlobalListener(listener, session);
-
-            MailboxPath inbox = MailboxPath.inbox(session);
             mailboxManager.createMailbox(inbox, session);
+            listener.clear();
+        }
+
+        @Test
+        void deleteMailboxShouldFireMailboxDeletionEvent() throws Exception {
+            assumeTrue(mailboxManager.hasCapability(MailboxCapabilities.Quota));
+
             mailboxManager.deleteMailbox(inbox, session);
 
             assertThat(listener.getEvents())
                 .filteredOn(event -> event instanceof MailboxListener.MailboxDeletion)
-                .isNotEmpty();
+                .hasSize(1)
+                .extracting(event -> (MailboxListener.MailboxDeletion) event)
```

Comment:
please use assertj assertions (with `satisfies` for example) and not allMatch so that we have better error reporting

Reply:
`satisfy` is not really satifying (no joke) as you get to assert on the full list.

Anyway we have a size assert so calling `.get(0)` should be OK